lib/elixir/lib/exception.ex

```diff
@@ -653,12 +662,18 @@ defmodule FunctionClauseError do
 end
 
 defmodule Code.LoadError do
-  defexception [:file, :message]
+  defexception [file: nil, message: nil, description: "could not load file"]
```

Comment:
There is still `message` and `description` in same definition.

On a side note, `[file: nil, message: nil, description: "could not load file"]` could be simplified as `[:file, :message, description: "could not load file"]`. :smiley: